transforms/csdl/preprocess_csdl_test_output.xml

```diff
@@ -59,6 +59,21 @@
       </EntityType>
       <EntityType Name="servicePrincipal" BaseType="graph.directoryObject" OpenType="true">
         <Property Name="appId" Type="Edm.String" />
+        <NavigationProperty Name="owners" Type="Collection(graph.directoryObject)">
+          <Annotation Term="Org.OData.Validation.V1.DerivedTypeConstraint" xmlns:edm="http://docs.oasis-open.org/odata/ns/edm">
```

Comment:
Validates both the `DerivedTypeConstraint` and the `NavigationRestrictions` annotations are added as needed